docs/modules/ROOT/pages/spring-cloud-kubernetes-discoveryserver.adoc

```diff
@@ -204,3 +204,7 @@ items:
 
 
 ----
+
+*Note* While the port is the same as Eureka's default (8761), there is no conflict despite the JSON
```

Comment:
ps wild guess this was chosen intentionally, like so custom servers can mount both endpoints and you only need to manage one discovery pod. If anyone knows the trivia why, do tell! cc @ryanjbaxter @spencergibb

Reply:
IDK why you would be using two different discovery servers

Reply:
on my side, it was a guess wondering why this is using the eureka port. I don't know the reason, so the important part was the question not my guess of the answer ;) If no one remembers, no worries.

Reply:
I know of at least one site trying to migrate from eureka to not eureka. That said, maybe the note is more trivia and calling attention to choosing the same well-known port as something else isn't helpful. I'll remove it!

Reply:
removed it!